readme.md

```diff
@@ -1,5 +1,39 @@
-# applib
-Cross-platform JavaScript library for Wikimedia apps
+# Library for common Javascript transforms used by both the Android and iOS Wikipedia apps
```

Comment:
Would you mind keeping the project title as the highest level element?

```markdown
# applib
Library for common Javascript transforms used by both the Android and iOS Wikipedia apps
```

Also consider rephrasing to include CSS: `..Javascript transforms and CSS...`.